x/ibc-account/keeper/relay.go

```diff
@@ -40,7 +40,13 @@ func (k Keeper) TryRegisterIBCAccount(ctx sdk.Context, sourcePort, sourceChannel
 		Data: salt,
 	}
 
-	// TODO: Add timeout height and timestamp
+	timeoutHeight := clienttypes.Height{
+		RevisionNumber: 2,
+		RevisionHeight: ^uint64(0),
```

Comment:
@colin-axner did you say that I can set `RevisionHeight` to `0` to stop `timeoutHeight` from being used as a timeout? I was getting a relayer error when I did this. 

I'm not sure what the best practice is here. @AdityaSripal maybe you have some thoughts also? :raised_hands:

Reply:
nvm updated